src/xtgeo/surface/_surfs_import.py

```diff
@@ -9,8 +9,8 @@
 logger = xtg.functionlogger(__name__)
 
 
-def from_grid3d(self, grid, subgrids, rfactor):
-    """Get surfaces from 3D grid, including subgrids"""
+def from_grid3d(grid, subgrids, rfactor):
+    """Get surfaces, subtype and order and from 3D grid, including subgrids"""
```

Comment:
Now there is an additional "and" here :P

Reply:
😬